contracts/modules/PoolController.sol

```diff
@@ -219,62 +219,71 @@ contract PoolController is
         }
     }
 
-    function release(bytes32 processId) 
+
+    function processPremium(bytes32 processId, uint256 amount) 
         external override
         onlyPolicyFlow("Pool")
     {
-        IPolicy.Policy memory policy = _policy.getPolicy(processId);
-        require(
-            policy.state == IPolicy.PolicyState.Closed,
-            "ERROR:POL-025:POLICY_STATE_INVALID"
-        );
-
         IPolicy.Metadata memory metadata = _policy.getMetadata(processId);
         IRiskpool riskpool = _getRiskpoolComponent(metadata);
-        riskpool.releasePolicy(processId);
+        riskpool.processPolicyPremium(processId, amount);
 
         uint256 riskpoolId = _riskpoolIdForProductId[metadata.productId];
-        IPolicy.Application memory application = _policy.getApplication(processId);
         IPool.Pool storage pool = _riskpools[riskpoolId];
-        uint256 collateralAmount = _collateralAmount[processId];
-
-        pool.sumOfSumInsuredAtRisk -= application.sumInsuredAmount;
-        pool.lockedCapital -= collateralAmount;
+        pool.balance += amount;
```

Comment:
It would be great to document the fields 
```
        uint256 capital;
        uint256 lockedCapital;
        uint256 balance;
``` 
in IPool to have a clear understanding what is what. It confuses me every time ;-)

Reply:
will do